src/client/gui/lib/providers.dart

```diff
@@ -7,6 +7,7 @@ import 'package:collection/collection.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:fpdart/fpdart.dart';
 import 'package:grpc/grpc.dart';
+import 'package:multipass_gui/platform/platform.dart';
```

Comment:
I recommend a relative import rather than an absolute one, and then it should be placed separately from those imports, in the same group as `import 'ffi.dart';` below.

```suggestion
import 'platform/platform.dart';
```